rs/boundary_node/ic_boundary/src/core.rs

```diff
@@ -345,6 +374,80 @@ pub async fn main(cli: Cli) -> Result<(), Error> {
         vec![Box::new(metrics_runner), Box::new(generic_limiter_runner)];
     runners.append(&mut registry_runners);
 
+    // HTTP Logs Anonymization
+    let tracker = (cli.misc.crypto_config)
+        .zip(cli.obs.obs_log_anonymization_canister_id)
+        .map(|(crypto_config, log_anonymization_cid)| {
+            let c = Arc::new(CryptoComponent::new(
+                &crypto_config,                          // config
```

Comment:
Which config will be used here?

Just for context, this config will have to be [the following](https://github.com/dfinity/ic/blob/master/ic-os/components/ic/generate-ic-config/ic.json5.template#L64-L71):
```
    crypto: {
        // The directory that should be used to persist crypto state.
        crypto_root: "/var/lib/ic/crypto",
        // The logic and metrics sockets for communicating with the crypto csp vault.
        // Shall match the values configured in
        // ic-crypto-csp.socket
        csp_vault_type: { unix_socket: { logic: "/run/ic-node/crypto-csp/socket", metrics: "/run/ic-node/crypto-csp/metrics" } },
    },
```
In particular, the vault type must be `unix_socket`.

For this to work, the [ic-crypto-csp.service](https://github.com/dfinity/ic/blob/master/ic-os/components/ic/ic-crypto-csp/ic-crypto-csp.service) (aka crypto vault) must be running. This service, running as user `ic-csp-vault`, is the only entity that is allowed to access the cryptographic (secret) key material (located in `/var/lib/ic/crypto`).

Communication with the crypto vault happens via the [ic-crypto-csp.socket](https://github.com/dfinity/ic/blob/master/ic-os/components/ic/ic-crypto-csp/ic-crypto-csp.socket). Only the mentioned user `ic-csp-vault` and members of the group `ic-csp-vault-socket` are allowed to access this socket. Besides the `ic-csp-vault` user itself, only the user `ic-replica` is a member of that group (see [this](https://github.com/search?q=repo%3Adfinity%2Fic+%2Fadduser.*ic-csp-vault-socket%2F&type=code)).

As which user will the API BN run? If it is not run as the `ic-replica` user (like the Orchestrator itself), then you'll have to add the respective user to the `ic-csp-vault-socket` group. I'm not aware of any SELinux rules around this (in fact there is an open [ticket](https://dfinity.atlassian.net/browse/NODE-854) to add such rules).

Reply:
Per @r-birkner - `ic-boundary` runs as `ic-replica`.